File: go/common/influxdb.go
package common

import (
	"github.com/influxdata/influxdb/client/v2"
)

func GetDBClient() client.HTTPClient {
	c, _ := client.NewHTTPClient(client.HTTPConfig{
		Addr:     "http://" + EncryptedCfg.Influxdb.Host + ":" + EncryptedCfg.Influxdb.Port,
		Username: EncryptedCfg.Influxdb.User,
		Password: EncryptedCfg.Influxdb.Password,
	})

	return c
}

File: python/src/common/influxdb.py
from common.configutil import encrypted_config
from influxdb import InfluxDBClient


##
# Gets a connection to an instance of InfluxDB.
#
# author: mjhwa@yahoo.com
##
def get_db_client():
  return InfluxDBClient(
    host=encrypted_config['influxdb']['host'], 
    port=encrypted_config['influxdb']['port'], 
    username=encrypted_config['influxdb']['user'], 
    password=encrypted_config['influxdb']['password']
  )